orc8r/tools/fab/BUILD.bazel

```diff
@@ -0,0 +1,18 @@
+# Copyright 2022 The Magma Authors.
+
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+load("@rules_python//python:defs.bzl", "py_library")
+
+py_library(
```

Comment:
Given that we are using `strtobool` also outside of Fabric we will also have to replace those eventually. Maybe it makes sense to already put this to a location not tied to Fabric now?

Reply:
We discussed this offline - I was in fear that the scope will explode here if we start to handle this for production and test logic in this issue. I suggest to do this in a follow up issue (and also evaluate there if we want to mix this at all or if we want to duplicate this).